### DISCUSSION TITLE
NEEDS ANIMATION

### DISCUSSION THREAD

**Discussion by trinhpxl** [keiwando/toonsquid#36] [Show and tell]
i did a [whole 13 hour stream](https://youtu.be/ZSnhtNje7Wc) of me tweening an animation ENTIRELY in toonsquid ! (except the 3d text) 
MAN this app is beautiful :) tysm dev for making this app and all ur hard work ! this is for a map, (multi animator project) with my two oc’s; vinx and trinh ! and i cant wait to post this to youtube when i get the chance ! 

![B9517FDC-AAA0-412E-9E2B-0750E816CE60](https://user-images.githubusercontent.com/115050959/198291686-141b4495-8fba-40f1-b891-362e73f91eaa.gif)

https://user-images.githubusercontent.com/115050959/198291694-96ddbd38-77ae-401e-bf39-755f232e3234.MOV

**Comment by LoneStarz** [comment]
OH MY GOD THAT LOOKS SO GOOD! Good job!!!!
